src/web/shared/Styles.ts

```diff
@@ -507,12 +508,35 @@ export function divePacketIconStyle(feature: Feature, animatedColor?: string) {
     })
 }
 
-export function driftPacketIconStyle(feature: Feature, animatedColor?: string) {
+/**
+ * Returns the drift icon index that should be displayed, given a drift speed.
+ * 
+ * @param driftSpeed Speed of the drift, in m/s
+ * @returns Index into Styles.driftArrowPngs, of the icon sthat should represent this drift
```

Comment:
needs return type {number}
Can we add a dash in between the type and the description -